src/Domain/Services/DraftOrders.php

```diff
@@ -110,7 +110,17 @@ public function register_draft_order_status( array $statuses ) {
 	 * @return array
 	 */
 	public function register_draft_order_post_status( array $statuses ) {
-		$statuses[ self::DB_STATUS ] = [
+		$statuses[ self::DB_STATUS ] = $this->get_post_status_properties();
+		return $statuses;
```

Comment:
Why is this different from the previous implementation?

Reply:
It's just abstracted so the draft post_status properties array don't have to be copied in multiple places.